packages/terra-clinical-result/tests/wdio/clinical-result-spec.js

```diff
@@ -0,0 +1,214 @@
+    before(() => {
+      browser.url('/#/raw/tests/terra-clinical-result/clinical-result/hidden-unit-observation');
+    });
+
+    Terra.it.validatesElement();
+  });
+
+  describe('bad data', () => {
+    before(() => {
+      browser.url('/#/raw/tests/terra-clinical-result/clinical-result/bad-data-observation');
+    });
+
+    Terra.it.validatesElement();
+  });
+
+  describe('null data', () => {
+    before(() => {
+      browser.url('/#/raw/tests/terra-clinical-result/clinical-result/null-observation');
+    });
+
+    Terra.it.validatesElement();
+  });
+
+  describe('interpretations', () => {
+    before(() => {
+      browser.url('/#/raw/tests/terra-clinical-result/clinical-result/interpretations-observation');
+    });
+
+    Terra.it.validatesElement();
+  });
+});
+
+Terra.describeViewports('FlowsheetResultCell', ['medium'], () => {
+  describe('default', () => {
+    before(() => {
+      browser.url('/#/raw/tests/terra-clinical-result/clinical-result/flowsheet-result-cell');
+    });
+
+    Terra.it.validatesElement();
+  });
+
+  describe('no data', () => {
+    before(() => {
+      browser.url('/#/raw/tests/terra-clinical-result/clinical-result/no-data-flowsheet-result-cell');
+    });
+
+    Terra.it.validatesElement();
+  });
+
+  describe('paddingStyles', () => {
+    before(() => {
+      browser.url('/#/raw/tests/terra-clinical-result/clinical-result/paddingstyle-flowsheet-result-cell');
+    });
+
+    Terra.it.validatesElement();
+  });
+
+  describe('multiple', () => {
+    before(() => {
+      browser.url('/#/raw/tests/terra-clinical-result/clinical-result/multiple-flowsheet-result-cell');
+    });
+
+    Terra.it.validatesElement();
+  });
+});
+*/
+Terra.describeViewports('ResultNameHeaderCell', ['medium'], () => {
+  describe('default', () => {
+    before(() => {
+      browser.url('/#/raw/tests/terra-clinical-result/clinical-result/result-name-header-cell');
+    });
+
+    Terra.it.validatesElement();
+  });
+
+  describe('paddingStyles', () => {
+    before(() => {
+      browser.url('/#/raw/tests/terra-clinical-result/clinical-result/paddingstyle-result-name-header-cell');
+    });
+
+    Terra.it.validatesElement();
+  });
+
+  describe('typeIndicators', () => {
+    before(() => {
+      browser.url('/#/raw/tests/terra-clinical-result/clinical-result/type-indicator-result-name-header-cell');
+    });
+
+    Terra.it.validatesElement();
+  });
+});
+/*
+Terra.describeViewports('ResultTimeHeaderCell', ['medium'], () => {
+  describe('default', () => {
+    before(() => {
+      browser.url('/#/raw/tests/terra-clinical-result/clinical-result/result-time-header-cell');
+    });
+
+    Terra.it.validatesElement();
+  });
+
+  describe('with date hidden', () => {
+    before(() => {
+      browser.url('/#/raw/tests/terra-clinical-result/clinical-result/result-time-header-cell-hide-date');
+    });
+
+    Terra.it.validatesElement();
+  });
+
+  describe('paddingStyles', () => {
+    before(() => {
```

Comment:
Lets remove all of these before hooks and use it blocks. This slightly
```suggestion
   describe('paddingStyles', () => {
      it('validates padding styles for time-header cell', () => {
          browser.url('/#/raw/tests/terra-clinical-result/clinical-result/paddingstyle-result-time-header-cell');
          Terra.validates.element();
      });
   })
```